src/ocm/services/ClustersService.ts

```diff
@@ -22,6 +23,27 @@ const ClustersService = {
     const fileName = contentHeader?.match(/filename="(.+)"/)?.[1];
     return { data, fileName };
   },
+
+  async update(cluster: Cluster, params: V2ClusterUpdateParams) {
+    if (ocmClient) {
+      params = ClustersService.updateClusterTags(cluster.tags, params);
+    }
+    return ClustersAPI.update(cluster.id, params);
+  },
+
+  updateClusterTags(
+    clusterTags: Cluster['tags'],
+    params: V2ClusterUpdateParams,
+  ): V2ClusterUpdateParams {
+    const tags = clusterTags?.split(',') || [];
+    if (tags?.includes(AI_UI_TAG)) {
```

Comment:
`tags` at this point would always be an array, no need for the `?` which makes the code harder to read.